torchao/kernel/intmm_triton.py

```diff
@@ -0,0 +1,349 @@
+import torch
+from torchao.kernel.autotuner import get_best_config_fn
+from torch._higher_order_ops.out_dtype import out_dtype
+from torch._dynamo import is_compiling as dynamo_is_compiling
+
+import triton
+import triton.language as tl
+import itertools
+import os
+int8_powers_of_two = [32, 64, 128, 256]
```

Comment:
do you envision people wanting to add more options here and for int8 kernel configs?

Reply:
Eventually, yes. Follow up here includes making it more extensible for other kernels. Adding support for mixed precision should help that.